README.md

```diff
@@ -820,6 +820,30 @@ Example (1000 loglines/second):
 cf set-env <YOUR_APP> LOG_RATELIMIT '1000'
 ```
 
+### Custom Runtime Metrics filtering
+
+For the third-party integrations explained above, in addition to the metrics collected by the agents, custom runtime metrics are provided via telegraf.
+This configuration also has a filtering mechanism that allows users to specify metrics they allow or deny for the vendor they are using.
+To be able to filter the emission of custom runtime metrics, users should provide list of prefixes of metrics they need to allow/deny using the environment variables listed below.
+
+Note: Custom database metrics cannot be filtered by name, to turn them off `APPMETRICS_INCLUDE_DB` environment variable should be set to false. 
+
+#### APM_METRICS_FILTER_DENY
+
+List of prefixes for the metrics to be denied. 
+
+For example to deny all metrics starting with **jetty** or **mx.runtime** the env var should be set to `jetty,mx.runtime`.
```

Comment:
"For example, to deny all metrics starting with **jetty** or **mx.runtime**, the environment variable should be set to `jetty,mx.runtime`."

Just a minor rewriting.

Reply:
updated